.github/workflows/golangci_lint.yml

```diff
@@ -15,10 +15,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: 1.19
+          go-version: "1.22"
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3.7.0
         with:
-          config-path: .golangci.yml
-          version: v1.49.0
-          skip-cache: true
\ No newline at end of file
+          version: v1.56.2
```

Comment:
why remove the config path with the custom file?

Reply:
https://github.com/dymensionxyz/dymension/pull/677/checks#step:4:1

The new version doesn't support those configuration options, or they were never supported, not sure which